Gordon360/Models/ViewModels/RecIM/ActivityPatchViewModel.cs

```diff
@@ -15,6 +15,7 @@ public class ActivityPatchViewModel
         public int? MaxCapacity { get; set; }
         public bool? SoloRegistration { get; set; }
         public string? Logo { get; set; }
+        public bool IsLogoUpdate { get; set; }
```

Comment:
I think this is a question for @Kevin-Chu58 but I think `Logo` is the only `patch` where is the value is null, it actually just removes the image.

Reply:
Yea see my later comment. I recommend adding post route that takes a nullable image. You wouldn't need a view model for that route.

Reply:
I see what you're saying and it makes sense. However, we had a slight UI implementation change where the image is actually completely disconnected from the rest of the updates. (This is for formatting reasons and random complications that the patch route has regarding the re-factoring of our form component)

I think with that in mind, it might be easier/better to make a general Route/Controller/Service for all of rec-im's images? Might need to do some brainstorming on what is the best implementation.

Reply:
I don't think I would make a centralized route/controller. Permissions will be managed differently for each upload case, so they should all have their own routes/controller methods. Not to mention that the changes are all affecting different types of resources, so the route should match that resource.

As for a central service/service method, I think that makes a lot of sense.